src/imagej/dims.py

```diff
@@ -467,3 +475,56 @@ def _to_ijdim(key: str) -> str:
         return ijdims[key]
     else:
         return key
+
+
+def _cal_axis_to_str(key) -> str:
+    """
+    Convert a CalibratedAxis class (e.g. net.imagej.axis.DefaultLinearAxis) to
+    a string.
+    """
+    cal_axis_to_str = {
+        jc.ChapmanRichardsAxis: "ChapmanRichardsAxis",
+        jc.DefaultLinearAxis: "DefaultLinearAxis",
+        jc.EnumeratedAxis: "EnumeratedAxis",
+        jc.ExponentialAxis: "ExponentialAxis",
+        jc.ExponentialRecoveryAxis: "ExponentialRecoveryAxis",
+        jc.GammaVariateAxis: "GammaVariateAxis",
+        jc.GaussianAxis: "GaussianAxis",
+        jc.IdentityAxis: "IdentityAxis",
+        jc.InverseRodbardAxis: "InverseRodbardAxis",
+        jc.LogLinearAxis: "LogLinearAxis",
+        jc.PolynomialAxis: "PolynomialAxis",
+        jc.PowerAxis: "PowerAxis",
+        jc.RodbardAxis: "RodbardAxis",
+    }
+
+    if key.__class__ in cal_axis_to_str:
+        return cal_axis_to_str[key.__class__]
+    else:
+        return "unknown"
+
+
+def _str_to_cal_axis(key: str):
+    """
+    Convert a string (e.g. "DefaultLinearAxis") to a CalibratedAxis class.
+    """
+    str_to_cal_axis = {
+        "ChapmanRichardsAxis": jc.ChapmanRichardsAxis,
+        "DefaultLinearAxis": jc.DefaultLinearAxis,
+        "EnumeratedAxis": jc.EnumeratedAxis,
+        "ExponentialAxis": jc.ExponentialAxis,
+        "ExponentialRecoveryAxis": jc.ExponentialRecoveryAxis,
+        "GammaVariateAxis": jc.GammaVariateAxis,
+        "GaussianAxis": jc.GaussianAxis,
+        "IdentityAxis": jc.IdentityAxis,
+        "InverseRodbardAxis": jc.InverseRodbardAxis,
+        "LogLinearAxis": jc.LogLinearAxis,
+        "PolynomialAxis": jc.PolynomialAxis,
+        "PowerAxis": jc.PowerAxis,
+        "RodbardAxis": jc.RodbardAxis,
+    }
+
+    if key in str_to_cal_axis:
+        return str_to_cal_axis[key]
+    else:
+        return None
```

Comment:
There's a cool one-liner `str_to_cal_axis.get(key, default=None)`. Or, for the bravest among us, `str_to_cal_axis.get(key)` can be used instead, because the default value for the `default` kwarg is `None`!

Reply:
I'm no longer using a dict to store the `CalibratedAxis` types. See 56ca2724a7f1a88fa30648d76c43427827a8aa49.